### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by aali1987**
Adding pandas package in the requirements inside the eman-deps installation package.

### CODE REVIEW

**Comment by aali1987** [comment]
@shadowwalkersb   @mstabrin   can you please check if I did it properly to add pandas package in the eman-deps installation routine.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 package:
     name: eman-deps-dev
-    version: '22.1'
+    version: '23.0'
```
Comment (shadowwalkersb) [inline recipe/meta.yaml]: Let's make this 24. I already have 23.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
 build:
     number: 4
```
Comment (shadowwalkersb) [inline recipe/meta.yaml]: This will need to be reset to 0.

**Review by shadowwalkersb — COMMENTED** [review]
I will make the changes I mentioned before.

**Pull request merged**